vscode-extension/src/extension.ts

```diff
@@ -110,5 +124,18 @@ export async function activate(context: ExtensionContext) {
     );
   });
 
-  context.subscriptions.push(client.start(), restartServerCommand);
+
+  spawn('java', resolveArgs)
+    .on('exit', code => {
+      if (code !== 0) {
+        const msg =  `Could not find Metals server artifact, ensure that metals.serverVersion setting is correct.
+                      Coursier resolve failed on:${artifact} with exit code:${code}.`.replace(/^(\s{2})+/gm, '');
```

Comment:
I don't know if there's a better way to do this in TS, but it wasn't obvious for me that this is meant to be like `.stripMargin` in Scala.

Reply:
I really miss `.stripMargin` in JavaScript.

The "trick" I usually adopt is:

```js
const msg = [
  'Could not find Metals server artifact, ensure that metals.serverVersion setting is correct.',
  `Coursier resolve failed on:${artifact} with exit code:${code}.`
].join('\n')
```